classport-agent/src/test/resources/classport-deps/classport-deps-list

```diff
@@ -0,0 +1,2 @@
+com.google.code.gson:gson:jar:2.11.0
+org.apache.commons:commons-lang3:jar:3.17.0
```

Comment:
We would later want this in JSON format so it is easily parseable. You could create another issue for this.
Same for `classport-deps-tree`.